.gitignore

```diff
@@ -1 +1,2 @@
 .work
+.DS_Store
```

Comment:
Please remove this from here and add it to your own gitignore see here how to do that https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer. Please delete all .DS_Store files from this PR.